test-cases/performance/perf-regression-predefined-throughput-steps.yaml

```diff
@@ -64,6 +64,5 @@ append_scylla_yaml:
   auto_snapshot: false
 
 use_placement_group: true
-use_prepared_loaders: false
 stress_image:
-  cassandra-stress: 'scylladb/cassandra-stress:3.13.0'
+  cassandra-stress: 'scylladb/cassandra-stress:3.17.0'
```

Comment:
@juliayakovlev IIUC the results with this client are lower than original, right?
If so, we what are we doing with it?

Reply:
**Mixed load test**
1. Scylla version `2024.1.0`, tablets **disabled**:
     - c-s version: 3.13.0
       Max throughput: [561309](https://docs.google.com/spreadsheets/d/1ybLBcPzhEUp_gPsgsKZ723pMijxS9Ln2VUBLBBCjujc/edit?gid=1514382928#gid=1514382928&range=H14)
    - c-s version: 3.17.0      
      Max throughput: [553260](https://docs.google.com/spreadsheets/d/1ybLBcPzhEUp_gPsgsKZ723pMijxS9Ln2VUBLBBCjujc/edit?gid=1514382928#gid=1514382928&range=H19)

2. Scylla version `2024.3.0~dev-20241106`, tablets **enabled**:
     - c-s version: 3.13.0
       Max throughput: [505753](https://docs.google.com/spreadsheets/d/1ybLBcPzhEUp_gPsgsKZ723pMijxS9Ln2VUBLBBCjujc/edit?gid=1514382928#gid=1514382928&range=H74)
    - c-s version: 3.17.0      
      Max throughput: [483284](https://docs.google.com/spreadsheets/d/1ybLBcPzhEUp_gPsgsKZ723pMijxS9Ln2VUBLBBCjujc/edit?gid=1514382928#gid=1514382928&range=H79)

**Read load test**
1. Scylla version `2024.1.0`, tablets **disabled**:
     - c-s version: 3.13.0
       Max throughput: [877512](https://docs.google.com/spreadsheets/d/1ybLBcPzhEUp_gPsgsKZ723pMijxS9Ln2VUBLBBCjujc/edit?gid=0#gid=0&range=H15)
    - c-s version: 3.17.0      
      Max throughput: [853437](https://docs.google.com/spreadsheets/d/1ybLBcPzhEUp_gPsgsKZ723pMijxS9Ln2VUBLBBCjujc/edit?gid=0#gid=0&range=H21)

2. Scylla version `2024.3.0~dev-20241106`, tablets **enabled**:
     - c-s version: 3.13.0
       Max throughput: [687654](https://docs.google.com/spreadsheets/d/1ybLBcPzhEUp_gPsgsKZ723pMijxS9Ln2VUBLBBCjujc/edit?gid=0#gid=0&range=H87)
    - c-s version: 3.17.0      
      Max throughput: [762589](https://docs.google.com/spreadsheets/d/1ybLBcPzhEUp_gPsgsKZ723pMijxS9Ln2VUBLBBCjujc/edit?gid=0#gid=0&range=H93)

Reply:
There's nothing in this change that is supposed to affect vnodes case.

We see here ~24k ops, in both directions, which seems a bit contradicting...

Can you share exact links to all those runs, so we can cross check it ?

Sound like we'll need to gather more results to understand what's going on

Reply:
@fruch @roydahan 
I re-ran the test with Scylla version `2024.3.0~dev-20241106`, tablets **enabled** and **c-s version 3.17.0**.

1. Max throughput in **mixed** load:

| Max throughput  | P99 read | link to run |
|-------------|----------------|-----|
| 541625      | 340.26         | https://argus.scylladb.com/tests/scylla-cluster-tests/8217747c-4120-436c-aac7-40c3c659f6c6 |
| 525913      | 916.98         |https://argus.scylladb.com/tests/scylla-cluster-tests/12342e92-d839-4aab-aeb5-fdf323453b4e |
| 559173      | 355.99         |https://argus.scylladb.com/tests/scylla-cluster-tests/e1ddb22d-7ad0-4080-9343-8558a6ba5772 |

In last 3 runs the max throughput is improved in mixed load + tablets enabled + c-s **3.17.0**.
Max throughput in run with c-s **3.13.0** is 505753 - see https://github.com/scylladb/scylla-cluster-tests/pull/9484#discussion_r1887015727


2. Max throughput in **read** load:

| Max throughput  | P99 read | link to run |
|-------------|----------------|-----|
| 677463      | 42.83         |https://argus.scylladb.com/tests/scylla-cluster-tests/b6cf3667-39ec-4662-9aee-22b3f3d2c8d5 |
| 782369      | 39.98         |https://argus.scylladb.com/tests/scylla-cluster-tests/5747dfe0-61c9-450a-a357-097a13c3186c |

The results with c-s **3.17.0** are not consistent

Reply:
The results with 3.13.0 are proved to be consistent, right?
Can you link please summarize them side by side and open a new issue to our cassandra-stress repo?
We can't switch to this release till we find out why it's not consistent.

Reply:
Also, I suggest 3 (or more consecutive runs on of unthrottled) on the same cluster to see if it's consistent on the same cluster.
(Best would be 3 runs with 3.13.0 and 3 runs with 3.17.0 on the same cluster.

Reply:
@roydahan will do when back